easybuild/easyconfigs/c/cget/cget-0.1.6-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'Bundle'
+
+name = 'cget'
+version = '0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('CMake', '3.10.2'),
+]
+
+exts_list = [
+    ('click', '6.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/click/'],
+        'checksums': [
+            'cc6a19da8ebff6e7074f731447ef7e112bd23adf3de5c597cf9989f2fd8defe9',  # click-6.6.tar.gz
```

Comment:
@jdemeul No need for the comment, and this can be collapsed to a single line; `eb --inject-checksums` will do this for you)